app/src/main/res/values/strings.xml

```diff
@@ -339,9 +339,10 @@
   </plurals>
   <string name="undo">Undo</string>
 
-  <plurals name="recent_pages">
+  <string name="recent_pages">Recent pages</string>
+  <plurals name="recent_pages" translatable="false">
```

Comment:
> should this be okay in locales that require plurals of zero, one, two, ...?

That's where I got the idea. For instance, in Arabic and Russian, this requires `%d` argument for plurals. As we do not care about actual numbers (in fact, we are not feeding any numbers for format args), that obviously does not work. Also, we do not show the last page / recent pages header when the amount is zero. So, no, this won't be an issue (hopefully:):

 - Zero --> not displated
 - Only one --> Last page
 - More than one --> Recent pages

The other are done in the second commit.